app/src/main/AndroidManifest.xml

```diff
@@ -7,11 +7,12 @@
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
-        android:supportsRtl="true"
-       >
-        <activity android:name=".MainActivity"
-            android:theme="@style/SplashTheme"
-            >
+        android:supportsRtl="true">
+
+        <activity
+            android:name=".MainActivity"
```

Comment:
Since it is the name of the splash, create an activity with name SplashActivity, apply to it the logic for splash screen and then navigate from SplashActivity to MainActivity